app/services/aapb/batch_ingest/pbcore_xml_ingester_behavior.rb

```diff
@@ -0,0 +1,114 @@
+# AAPB::BatchIngest::PBCoreXMLItemIngesterBehavior
+# This module is to DRY up common methods  between PBCoreXMLItemIngester and
+# PBCoreXMLInstantiationReset, but also for any future custom ingester that
+# needs to behave mostly like PBCoreXMLItemIngester.
+module AAPB
+  module BatchIngest
+    module PBCoreXMLIngesterBehavior
+      # Common private methods ingester classes.
+      private
+
+        def batch_item_is_asset?
```

Comment:
yeah, a little risky, but too low to block on this i think. It's not new code, it was just pulled out into a behavior module to it could be included in 2 places. So it's been working without error. But we can make a new ticket for it if needed.

Reply:
nah that's fine, we don't have any records that are themselves about pbcore